Utils/azure-explorer-common/src/com/microsoft/tooling/msservices/serviceexplorer/azure/webapp/WebAppPropertyMvpView.java

```diff
@@ -4,7 +4,7 @@
 import com.microsoft.azuretools.core.mvp.ui.webapp.WebAppProperty;
 
 public interface WebAppPropertyMvpView extends MvpView {
```

Comment:
Rename the interface name

Reply:
Will rename to `WebAppBasePropertyMvpView `.